projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -219,16 +128,17 @@ export function NewsletterAccessRadioButtons( {
 								htmlFor={ `editor-post-${ key }-${ instanceId }` }
 								className="editor-post-visibility__label"
 							>
-								{ accessOptions[ key ].label }
+								{ accessOptions[ key ].label }{ ' ' }
+								{ key !== accessOptions.everybody.key && (
+									<>
+										{ ' ' }
+										{ sprintf(
+											'(%1$s)',
+											getReachForAccessLevelKey( key, emailSubscribers, paidSubscribers )
+										) }
```

Comment:
This whole section with multiple empty strings, `sprintf` and whatnot felt complicated to grok at glance but I don't have great alternative. 🤔  I had something like this in mind:

```js
const accessLabel = accessOptions[ key ].label;
const reach = key !== accessOptions.everybody.key ? getReachForAccessLevelKey( key, emailSubscribers, paidSubscribers ) : false;
```
```jsx
<label>
    { accessLabel }
    { subscriberReach && ` (${ subscriberReach })` }
</label>
```